edx_repo_tools/codemods/django3/github_actions_modernizer_django.py

```diff
@@ -0,0 +1,48 @@
+"""
+Django Matrix Modernizer for Github Actions CI
+"""
+from copy import deepcopy
+
+import click
+
+from edx_repo_tools.utils import YamlLoader
+
+DJANGO_VERSIONS = ['2.2', '3.0', '3.1', '3.2']
+
+
+class GithubCIDjangoModernizer(YamlLoader):
+    def __init__(self, file_path):
+        super().__init__(file_path)
+        self.yml_instance.default_flow_style = None
+        self.yml_instance.indent(mapping=2, sequence=2, offset=0)
+
+    def _update_matrix(self):
+
+        section_key = None
+
+        for key in ['build', 'tests', 'run_tests']:
+            if key in self.elements['jobs']:
+                section_key = key
+
+        if 'django-version' in self.elements['jobs'][section_key]['strategy']['matrix']:
```

Comment:
this should be a little dynamic in identifying sub-elements of a matrix if any of the matrix items matches `django` we should select that element and update its sequence (i.e adding new Django versions for testing)
for example in some places we've `toxenv` that sets up Django versions as matrix items
`toxenv: [django22, quality, docs, pii_check]`